crates/transcode/src/lib.rs

```diff
@@ -124,10 +124,26 @@ use anyhow::{
     Context,
     Result,
 };
-use ink_metadata::{
-    ConstructorSpec,
+pub use ink_metadata::{
```

Comment:
Why not just reexport the whole crate i.e. `pub use ink_metadata`?

Gives us namespacing and means we don't have to add/remove types.

Reply:
Reexporting the whole crate works for me

Reply:
If you update the PR today we can add it to an upcoming release.

Reply:
I can do that